bin/.travis/prepare_behat.sh

```diff
@@ -1,27 +1,15 @@
-#!/bin/sh
+#!/bin/bash
 
-# File for setting up system for behat testing, just like done in DemoBundle's .travis.yml
-
-# Change local git repo to be a full one as we will reuse it for composer install below
-git fetch --unshallow && git checkout -b tmp_ci_branch
-export BRANCH_BUILD_DIR=$TRAVIS_BUILD_DIR TRAVIS_BUILD_DIR="$HOME/build/ezplatform"
-
-# Checkout meta repo, use the branch indicated in composer.json under extra._ezplatform_branch_for_behat_tests
 EZPLATFORM_BRANCH=`php -r 'echo json_decode(file_get_contents("./composer.json"))->extra->_ezplatform_branch_for_behat_tests;'`
+EZPLATFORM_BRANCH="${EZPLATFORM_BRANCH:-master}"
+PACKAGE_BUILD_DIR=$PWD
+EZPLATFORM_BUILD_DIR=${HOME}/build/ezplatform
 
-cd "$HOME/build"
-
-git clone --depth 1 --single-branch --branch "$EZPLATFORM_BRANCH" https://github.com/ezsystems/ezplatform.git
-cd ezplatform
+echo "> Cloning ezsystems/ezplatform:${EZPLATFORM_BRANCH}"
+git clone --depth 1 --single-branch --branch "$EZPLATFORM_BRANCH" https://github.com/ezsystems/ezplatform.git ${EZPLATFORM_BUILD_DIR}
+cd ${EZPLATFORM_BUILD_DIR}
 
-#if [ "$REST_TEST_CONFIG" != "" ] ; then
-#    echo "> Fixing security.yml for REST functional tests"
-#    sed -i "s@#        ezpublish_rest:@        ezpublish_rest:@" app/config/security.yml
-#    sed -i "s@#            pattern: ^/api/ezp/v2@            pattern: ^/api/ezp/v2@" app/config/security.yml
-#    sed -i "s@#            stateless: true@            stateless: true@" app/config/security.yml
-#    sed -i "s@#            ezpublish_http_basic:@            ezpublish_http_basic:@" app/config/security.yml
-#    sed -i "s@#                realm: eZ Publish REST API@                realm: eZ Publish REST API@" app/config/security.yml
-#fi
+# Update docker to supported version
+${EZPLATFORM_BUILD_DIR}/bin/.travis/trusty/update_docker.sh
```

Comment:
Note to reviewers: moved this call outside of script in meta-repository as this:
- potentially depends on `.travis.yml` setup used for current build,
- allows to reuse meta-repository script for behat setup on meta-repository itself (for branches and PRs opened there).

So one more line in dependent package setup, but I feel like this is the best choice.

Reply:
yes, this needs to go, we need to control this in meta repo.

And we most likely don't need it anymore, as docker was updated among many other things on travis on Tuesday.